### PR TITLE
Make all supported update scripts compatible with strict mode

### PR DESCRIPTION
**Pull request by ganoze**
## Description

Fix upgrade scripts to handle MariaDB 10.3 along with strict mode.

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [X] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [X] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Upgrade from Centreon 2.8. Everything should work as expected.

## Checklist

- [X] I followed the **coding style guidelines** provided by Centreon
- [X] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [X] I have commented my code, especially **hard-to-understand areas** of the PR.
- [X] I have made corresponding changes to the **documentation**.
- [X] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Comment by ganoze** [comment]
Tested and working with MariaDB 10.3 with strict mode enabled.

**Comment by lgcosta** [comment]
> Tested and working with MariaDB 10.3 with strict mode enabled.

Hi @ganoze 
Tested with filled data? for example, I took cases where the `enum` field was as an integer and strict-mode requires that it be as a string otherwise it generates an error.

**Comment by ganoze** [comment]
> Tested with filled data?

Yes indeed. I tested with a medium-sized machine with most modules installed. I will test with a bigger machine now.

**Comment by ganoze** [comment]
Good, successful upgrade on a big setup !

**Comment by adr-mo** [comment]
I though I had click approved my bad lol

File: www/install/php/Update-19.10.0-beta.3.php [hunk 1]
```diff
@@ -145,13 +151,16 @@
...
+} finally {
+    $pearDB->query('SET SESSION innodb_strict_mode=ON');
 }
 
 
 /**
  * Add columns to manage engine & broker restart/reload process
  */
 
+$pearDB->query('SET SESSION innodb_strict_mode=OFF');
```
Comment (callapa) [inline www/install/php/Update-19.10.0-beta.3.php:163]: Please insert the following requests in a "try - catch" block.

**Pull request merged**